app.json

```diff
@@ -8,12 +8,98 @@
     "APP_KEY": {
       "required": true
     },
-  "buildpacks": [
-    {
-      "url": "heroku/php"
+    "APP_LOG": "errorlog",
+    "APP_URL": "https://ds-rogue-staging.herokuapp.com/",
+    "buildpacks": [
+      {
+        "url": "heroku/php"
+      },
+      {
+        "url": "heroku/nodejs"
+      }
+    ],
+    "AWS_ACCESS_KEY_ID": {
+      "required": true
+    },
+    "AWS_REGION": {
+      "required": true
+    },
+    "AWS_SECRET_KEY": {
+      "required": true
+    },
+    "BLINK_URL": {
+      "required": true
+    },
+    "BLINK_USERNAME": {
+      "required": true
+    },
+    "BLINK_PASSWORD": {
+      "required": true
+    },
+    "CACHE_DRIVER": {
+      "required": true
+    },
+    "DB_DATABASE": {
+      "required": true
+    },
+    "DB_HOST": {
+      "required": true
+    },
+    "DB_PASSWORD": {
+      "required": true
+    },
+    "DB_USERNAME": {
+      "required": true
+    },
+    "DS_ENABLE_BLINK": "true",
```

Comment:
I think so? This app inherits from `rogue-staging` which is connected to blink staging so I figured it would just allow us to make sure the integration is working. However, I could see a case for only adding it in when we need to test that.

Reply:
Yeah, I could see that being helpful. My only concern would be polluting the data in Customer.io staging with things from review apps... maybe something we can switch on case-by-case for specific PRs?